tests/telemetry_test.py

```diff
@@ -3431,27 +3449,27 @@ def test_stores_default_stats(self, metrics_store_put_doc):
 
         metrics_store_put_doc.assert_has_calls([call({
             "name": "segment-replication-stats",
-            "shard_id": "[so][0]",
-            "target_node": "node-1",
-            "target_host": "127.0.0.1",
+            "shard_id": "[logs-211998][6]",
```

Comment:
nit: Is `shard_id` change needed here ?

Reply:
@dreamer-89 Thanks a lot for your review! 👏 The change is indeed not necessary, I just updated the test with a real API response from my recent performance test. I think it's fine to keep the changes.